docs/docs/mdx/writing-pages.md

```diff
@@ -28,7 +28,7 @@ date: 2019-01-29
 # Hello, world!
 ```
 
-Which can then be [queried with GraphQL](/docs/querying-with-graphql/):
+Which can then be [queried with GraphQL](/docs/page-query/):
```

Comment:
Given this redirect: https://github.com/gatsbyjs/gatsby/blob/30437865257c21498b6da0df026e67d2983bfa2c/www/gatsby-node.js#L368-L372

Should we change link to `/docs/graphql-concepts/`?

Reply:
Good call. Didn't think to check the gatsby-node file. Will update to `/docs/graphql-concepts/`

Reply:
I navigated to old url directly and saw that I was redirected and then found this code (I wouldn't start by looking there :P)

Reply:
Oh good, thanks for the reassurance! I changed it to '/docs/graphql-concepts/' and commited again.